lib/elixir_auth_github.ex

```diff
@@ -126,8 +128,7 @@ defmodule ElixirAuthGithub do
 
     # transform map with keys as strings into keys as atoms!
     # https://stackoverflow.com/questions/31990134
-    atom_key_map = for {key, val} <- user, into: %{}, do: {String.to_atom(key), val}
-    {:ok, atom_key_map}
+    _atom_key_map = for {key, val} <- user, into: %{}, do: {String.to_atom(key), val}
```

Comment:
Can't say I understand why this is being changed. 
Generally the preference is to return an `{:ok, value}` tuple 
So the calling function can pattern match on the status `:ok` or `:error` 💭

Reply:
You're right! reverted.